### DISCUSSION TITLE
Subscribe to store: is it forbidden or not?

### DISCUSSION THREAD

**Discussion by rvale01** [ngrx/platform#4114] [Q&A]
### Information

Hi everyone!
I've started to use ngrx recently and I'm trying to understand how it works. 
While reading in the documentation, I noticed that there are many examples where after a selector, subscribe is used. 
However, by using eslint-plugin, there is a rule which says 
"`Store` subscription is forbidden. Use the `async` pipe instead.eslint(@ngrx/no-store-subscription)"

So, is it actually forbidden or is it more of a suggestion? 
If so, why is that?

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by userlerueda** [comment]
I have the same concern here. What if the values that I need are not for the HTML but for the TS file? I agree that in my opinion it may be a suggestion more than something else.

**Comment by david-shortman** [comment]
It's a big topic, but at a high level, you can pretty broadly avoid manually `subscribe`ing to the global store, and pretty much **any** Observable, by:
- in templates, using `async` or consuming state as Signals
- for anything else, isolate processing of data in effects which manage a subscription for you
